backend/src/main/java/com/hackner/musicompass/discogsapi/model/DiscogsArtist.java

```diff
@@ -6,6 +6,8 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.util.Objects;
+
```

Comment:
I don't see a reference in the code to "Objects", you can probably remove that. 
In general any IDE should show you with a warning when you have unused imports, so always try to go over all the files you worked on in your current commit if you have some of those. (Most of these are not intentional anyway)

Reply:
deleted. thx